.circleci/config.yml

```
@@ -45,6 +45,15 @@ commands:
           name: Build NuGet Package
           command: dotnet pack src/SignalFx.LambdaWrapper/SignalFx.LambdaWrapper.csproj -c Release -o bin/package 
 
+  copy_nuget_for_release:
+    steps:
+      - run:
+          name: Create artifacts directory
+          command: mkdir artifacts
+      - run:
+          name: Copy NuGet packages to artifacts directory
+          command: cp -a bin/package/. artifacts/
```

```suggestion 
      - run:
          name: Copy NuGet packages to artifacts directory
          command: |
            mkdir -p artifacts
            cp -a bin/package/. artifacts/
```

1. I think we can join those two steps
2. `mkdir -p` does not fail if the dir exists